configuration/concepts/commons.csv

```diff
@@ -2,4 +2,11 @@ Uuid,Void/Retire,Same as mappings,Fully specified name:en,Short name:en,Fully sp
 67218dc9-aee7-40d1-ba78-d29dd2731333,,CIEL:1362,One plus,+,Uno más,+,,Misc,N/A,,,,,,,,,,,
 55800eb1-4e0f-4a75-a12b-aa1e10c64e81,,CIEL:1363,Two plus,++,Dos más,++,,Misc,N/A,,,,,,,,,,,
 17327c32-0a60-4f76-85b5-24379fc72ad5,,CIEL:1364,Three plus,+++,Tres más,+++,,Misc,N/A,,,,,,,,,,,
-f0853fb9-5541-4e13-b79a-e3b438a629cb,,CIEL:1365,Four plus,++++,Cuatro más,++++,,Misc,N/A,,,,,,,,,,,
\ No newline at end of file
+f0853fb9-5541-4e13-b79a-e3b438a629cb,,CIEL:1365,Four plus,++++,Cuatro más,++++,,Misc,N/A,,,,,,,,,,,
+a11a2309-3600-4ea8-957c-18591654ad37,,,Both,Both,Ambos,Ambos,,Misc,N/A,,,,,,,,,,,
+23e59dfe-0690-4826-8059-efa8f8b13d73,,,Right,Right,Derecho,Derecho,,Misc,N/A,,,,,,,,,,,
+8192d078-9333-4672-86f1-96c07b12188d,,,Left,Left,Izquierdo,Izquierdo,,Misc,N/A,,,,,,,,,,,
+b50bd43f-07a2-45e2-aee2-20bb16359057,,,Months,Months,Meses,Meses,,Misc,N/A,,,,,,,,,,,
```

Comment:
This concept already exists in Bamni-core. No need to recreate it here

Reply:
Removed

Reply:
Done